frontend/pages/play/members.vue

```diff
@@ -242,13 +242,20 @@ onMounted(() => {
         </div>
       </div>
       <div class="flex w-full justify-end">
-        <button
-          :disabled="!isReady"
-          @click="modalIsOpen = true"
-          class="w-72 rounded-lg bg-sc-green py-6 text-center text-4xl font-bold text-sc-black drop-shadow-sc-shadow hover:bg-sc-green-400 disabled:bg-sc-black-400 disabled:text-sc-white"
-        >
-          {{ $t("join_room.add_members_ready") }}
-        </button>
+        <UPopover mode="hover" :disabled="isReady">
+          <template #panel>
+            <p class="w-80 p-3">
+              {{ $t("join_room.product_owner_or_scrum_master_required") }}
```

Comment:
denke es wäre schöner, wenn "weitermachen" getrennt wird
![image](https://github.com/user-attachments/assets/a5194917-d1c7-481c-99c4-5c7e38c1c6ab)

Reply:
ist nur leider nicht richtig, schau in dein Wörterbuch: das ist eine trennbare Zusammensetzung (weil man sagen kann "ich mache weiter" im Gegensatz zB zu widersprechen), da die Betonung auf dem Adverb liegt schreibt man das zusammen